meta/case.rb

```diff
@@ -155,3 +155,29 @@
     end
   RUBY
 end
+
+Mutant::Meta::Example.add :case do
```

Comment:
Would you like to tag sorbet related specs somehow?

Reply:
Yes, lets move this to `meta/sorbet.rb` for the moment.

Reply:
Sure!